examples/messages/fetch-message-log/main.bal

```diff
@@ -18,14 +18,16 @@ import ballerina/io;
 import ballerina/os;
 import ballerinax/twilio;
 
+configurable string apiKey = os:getEnv("API_KEY");
+configurable string apiSecret = os:getEnv("API_SECRET");
 configurable string accountSid = os:getEnv("ACCOUNT_SID");
-configurable string authToken = os:getEnv("AUTH_TOKEN");
 
 // Twilio configurations
 twilio:ConnectionConfig twilioConfig = {
     auth: {
-        accountSid,
-        authToken
+        apiKey,
+        apiSecret,
+        accountSid
```

Comment:
Shall we reorder ?
```suggestion
        accountSid,
        apiKey,
        apiSecret
```

Reply:
Any reason behind this?

Reply:
As discussed offline this approach suites the current design. Hence +1.